src/screens/Library/components/InstallModal/index.css

```diff
@@ -82,6 +82,12 @@
   margin: 12px;
 }
 
+.dlcList {
+  padding: 12px 0 18px;
+  display: flex;
+  flex-direction: column;
```

Comment:
Maybe we need a max-height here with a scroll? not sure but I imagine some games have a lot of DLCs (which causes the original issue) making the install modal too tall maybe even overflow the screen.

Reply:
yes, makes sense. I'll change it 👍🏽

Reply:
done